examples/data-sources/gitlab_group_variable/data-source.tf

```
@@ -0,0 +1,11 @@
+data "gitlab_group_variable" "foo" {
+  group = "my/example/group"
+  key   = "foo"
+}
+
+# Using an environment scope
+data "gitlab_group_variable" "bar" {
+  project           = "my/example/group"
```

```suggestion
  group = "my/example/group"
```

I think this is supposed to be group 😄